pkgs/by-name/fu/fuzzel/package.nix

```
@@ -54,14 +56,15 @@ stdenv.mkDerivation (finalAttrs: {
     tllist
     fcft
   ] ++ lib.optional enableCairo cairo
-    ++ lib.optional pngSupport libpng;
+  ++ lib.optional pngSupport libpng
+  ++ lib.optional (svgBackend == "librsvg") librsvg;
```

```suggestion
  ++ lib.optional (svgSupport && svgBackend == "librsvg") librsvg;
```
Similar to the above

---

Fixed